tests/unit/services/gmail/test_gmail_service.py

```
@@ -218,3 +218,65 @@ def test_subject_with_special_characters(self):
+            {"name": "From", "value": "Updates <updates@mail.service.example.com>"},
+        ]
+        name, email = _get_sender_info(headers)
+        assert email == "updates@mail.service.example.com"
+        assert "@" in email
+
+    def test_email_extraction_strips_whitespace(self):
+        """Email extraction should handle headers with extra whitespace."""
+        headers = [
+            {"name": "From", "value": "  Newsletter  < news@example.com >"},
+        ]
+        name, email = _get_sender_info(headers)
```

Ruff (0.14.10)

Unpacked variable `name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)